### PR TITLE
EXSWHTEC-172 - Hip graph launching and execution tests

### PR DESCRIPTION
**Pull request by marko-veniger**
Implement positive and negative unit test for the following API's
- hipGraphLaunch
- hipGraphUpload
- hipGraphExecDestroy
- hipGraphExecUpdate

### CODE REVIEW

File: catch/unit/graph/hipGraphUpload.cc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include <hip_test_common.hh>
+#include <hip_test_checkers.hh>
+
+#if 0
```
Comment (chrispaquot) [inline catch/unit/graph/hipGraphUpload.cc]: huh?

Comment (nives-vukovic) [reply catch/unit/graph/hipGraphUpload.cc]: @chrispaquot Code is merged with develop and modified accordingly. Code disabling has been removed

**Review by chrispaquot — APPROVED** [review]

**Comment by rorake** [comment]
@marko-veniger 
Will you please resolve the merge conflicts for this PR?

**Comment by nives-vukovic** [comment]
@rorake Conflicts are resolved.

**Comment by mangupta** [comment]
PR has been merged into develop branch via commit 133521b22f95d577391544a39d5c3f88ff98b5d7

**Pull request closed without merging**